### DISCUSSION TITLE
Frequent "Something went really wrong" error when pasting images

### DISCUSSION THREAD

**Discussion by mrwweb** [community/community#40375] [Projects and Issues]
### Select Topic Area

Bug

### Body

I've been getting the error "Something went really wrong, and we can’t process that file. Try again" _constantly_ for the last month or so when trying to drag/paste/upload an image to Issue comments. I'd say it doesn't work 90%+ but then magically an upload will work sometimes.

I'm on Windows 10 and primarily use Firefox. It seems that this may somehow be tied to the Windows Snip & Sketch tool which I most frequently use to take screenshots. For instance, it seems if I take a screenshot with this tool and paste it into Cloudup and then copy and paste _that_ image, it works. However, if I save the Snip & Sketch screenshot (either renaming the file or not), I still can't drag or upload an image into an issue, so it's not just about pasting.

This really feels like a bug, but I'm not sure how to help troubleshoot it or get this information to the right person.

**Comment by ossanna16** [comment]
Hi there @mrwweb and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
